composer.json

```diff
@@ -41,6 +42,25 @@
         "psr-4": {
         }
     },
+    "autoload-dev": {
+        "psr-4": {
+            "PhpList\\BaseDistribution\\Tests\\": "Tests/"
+        }
+    },
+    "scripts": {
```

Comment:
Interesting -- this will automatically copy phplist-4 core into the web root directory following installation  and updates? Is this a long-term or short term solution for the base-distribution?